lib/cli/src/commands/create_exe.rs

```diff
@@ -1443,31 +1454,64 @@ mod http_fetch {
             }
         };
 
+        // Test if file has been already downloaded
         if let Ok(mut cache_path) = super::get_libwasmer_cache_path() {
-            match std::fs::read_dir(&cache_path).and_then(|r| {
+            let paths = std::fs::read_dir(&cache_path).and_then(|r| {
                 r.map(|res| res.map(|e| e.path()))
                     .collect::<Result<Vec<_>, std::io::Error>>()
-            }) {
-                Ok(mut entries) => {
+            });
+
+            if let Ok(mut entries) = paths {
+                entries.retain(|p| p.to_str().map(|p| p.ends_with(".tar.gz")).unwrap_or(false));
+
+                // create-exe on Windows is special: we use the windows-gnu64.tar.gz (GNU ABI)
+                // to link, not the windows-amd64.tar.gz (MSVC ABI)
+                if target_triple.operating_system == OperatingSystem::Windows {
+                    entries.retain(|p| {
+                        p.to_str()
+                            .map(|p| p.contains("windows") && p.contains("gnu64"))
```

Comment:
Does `create-exe` support 32-bit Windows?

Reply:
No, we don't support that.

Reply:
Awesome!